app/services/projects/copy/wiki_dependent_service.rb

```diff
@@ -69,9 +69,6 @@ def copy_wiki_pages(_params)
     end
 
     def copy_wiki_page(source_page, new_parent_id)
-      # Skip pages without content
-      return if source_page.content.nil?
```

Comment:
Should we have `return if source_page.text.nil?` instead? Admittedly I think, it makes no sense to not copy empty pages, so I'm fine with not having the guard clause here.

Reply:
I think the level this guard was on was data consistence. So it did not guard against the page not having text (then it would have to have checked `source_page.content.text.present?`). Rather, it checked that there was a `content` object associated to the page. 

Such a case can no longer occur so I think it safe to simply remove the guard.